api/handler/login_handelr.go

```diff
@@ -15,18 +16,21 @@ LoginHandler is a function for gin to handle login api
 func LoginHandler(c *gin.Context) {
 	authMiddleware, err := middleware.NewAuthMiddleware()
 	if err != nil {
-		log.Printf("JWT Error:" + err.Error())
+		logs.Warning.Printf("JWT Error:" + err.Error())
 		c.Status(500)
 	}
 
 	studentID := c.PostForm("studentID")
 	password := c.PostForm("password")
 
+	fmt.Println("password", password)
+	fmt.Println("studentID", studentID)
```

Comment:
Are these `Println`s here left intentionally?

Reply:
This is old data.....